### ISSUE TITLE
select many items in bp, symbols, refs view

### ISSUE THREAD

**Issue by lstolcman** [x64dbg/x64dbg#1304]
1. select many items with shift
 this is currently doable in symbols view only:
![vcxrxxq 1](https://cloud.githubusercontent.com/assets/4583553/20684488/69bf41ce-b5b0-11e6-934c-2dc225ee05a1.png)

and in symbols modules:
![aa3tax4 1](https://cloud.githubusercontent.com/assets/4583553/20684575/dc3fe122-b5b0-11e6-8715-4c3e682b1c01.png)

should be also doable in breakpoints:
![y5e9x3p 1](https://cloud.githubusercontent.com/assets/4583553/20684600/f705c116-b5b0-11e6-9055-ee17296d5e46.png)

may be useful to implement it in references->strings and references->intermodular calls

also:
2. allow ctrl+a (select all) - select all items - not available but should be in these above cases

why? this is usable in case of adding/removing breakpoints groups

**Comment by mrexodia** [comment]
I agree with you. We are open to pull requests and if you're interested take a look at it, should be easy to implement.

**Comment by mrexodia** [comment]
@kkthx I think this issue can be closed. Can you verify?

**Comment by kkthx** [comment]
Hello,
About first issue (select many items by shift) - it is fixed
About second - ctrl-a (select) all - working in "breakpoints" window, but not in "Symbols" window

**Comment by justanotheranonymoususer** [comment]
Multiple items can be selected in symbols and breakpoints view, but not in references. It would be great if it could be added there too.
